### DISCUSSION TITLE
English Text Epilogue/Finale Credits

### DISCUSSION THREAD

**Discussion by Wandering-Heiho** [dackst/nayuta#11] [Show and tell]
Howdy, sorry for the silence on my end regarding the movie file edits, I've just been a little busy. After some time, I feel I have finally gotten this to a level of quality I am content with.

With regards to removing the original text, I only did that for three sections: The text at the beginning (the one with PSP Playstation Portable underneath), the text immediately after that one (translated to Cast), and the Nayuta no Kiseki logo near the very end. I ended up not trying to remove the original text for character names and VAs listed, as I was worried about making sure the details behind it were not damaged/created artifacts. Instead, I had the English text appear over a backing that would cover up the Japanese text. Here is an example.

https://user-images.githubusercontent.com/73907314/111728469-51b5fe00-8843-11eb-9221-a0d6234058f0.mp4

So that you can have an idea of what the full thing currently looks like, I've placed a URL here.
https://youtu.be/952mWzgtjF0

**If you want any changes to be made, let me know.**

If you are fine with the current result, here is the .pmf file I tested to make sure it worked in-game.
[Nayuta_no_Kiseki-English_Ending1.zip](https://github.com/dackst/nayuta/files/6168479/Nayuta_no_Kiseki-English_Ending1.zip)

**Comment by dackst** [comment]
Don't worry about it, take as long as you think you need.

I like the rest, but I have two things about the character names:

1. If we're going to show both the Japanese text and English text (which I think is reasonable, foreign names on Wikipedia will often include some equivalent in the current language, as well as the name how it is originally written), how about giving some more time so that the Japanese text could still be read for those who can? Maybe like at least increasing the delay between when the Japanese first appears and then when the English appears over it.

2. Can the size of the backing behind the English text be increased? There are some names where the edges of the Japanese still peek out under the English.

**Reply by Wandering-Heiho** [reply]
1. Sure, I'll see about increasing the delay.

2. Yep, I should be able to easily enlarge the backing.

**Reply by dackst** [reply]
So I was thinking, maybe you wouldn't have to worry about the delay or the backing if you made the English text smaller and put it next to the Japanese instead of on top of it. For the overlapping route, another possibility might be to make the English appear before any hint of the original text appears at all, but I don't know how much that blue particle effect that starts a little offset from the final location of each name would complicate things.

But these are just ideas, do whatever you think you would be satisfied with.

**Reply by Wandering-Heiho** [reply]
Having the text next to the Japanese text was an idea I had early on, but I scrapped it because I didn't feel there would be enough space to the right of the Japanese text. I also didn't want to put the new text to the left, as I wanted to avoid covering up other elements of the ending which could lead to contrast problems (hello glowing yellow orb).

I've tested out enlarging the backings and adding a small delay to the English text before it covers up the original text. Since the delay would last until all of the onscreen text appeared, I tried out having all of the names change at the same time (mainly because I felt keeping the fade in offsets I originally had would be distracting).

https://user-images.githubusercontent.com/73907314/111894900-1d7a4300-89e5-11eb-9aaa-155a17c219c9.mp4

Like last time, this is what the full thing currently looks like.
https://youtu.be/AXIF4FAk8ls

**Reply by dackst** [reply]
Yeah, you're right, I just tried using the same pixel font from the game to add *Song Maiden Erisette* to the right of the Japanese, and there's just barely *not* enough space:
![NPJH50625_00012](https://user-images.githubusercontent.com/35816592/111904840-3869a900-8a1f-11eb-98c8-af8d635bb2e3.png)

It makes sense that it could be distracting to have several different things changing on the screen at the same time. Anyway, now that the Japanese isn't replaced *immediately* right after it appears, and that it is fully covered when it is replaced, I'm satisfied with it.

**Reply by Wandering-Heiho** [reply]
Glad to hear.

Here is the .pmf file.
[Nayuta_no_Kiseki-English_Ending1.zip](https://github.com/dackst/nayuta/files/6177722/Nayuta_no_Kiseki-English_Ending1.zip)
